### PR TITLE
HAAR-986: Amended Users Group endpoint

### PR DESCRIPTION
**Pull request by ravmoj**
HAAR-986:Amended Users Group endpoint from Auth to Manage Users endpoint

### CODE REVIEW

File: integration-tests/mockApis/manageusers.js [hunk 1]
```diff
@@ -470,6 +470,117 @@ const stubChangeGroupName = () =>
...
+      { roleCode: 'GLOBAL_SEARCH', roleName: 'Global Search', automatic: true },
+      { roleCode: 'LICENCE_RO', roleName: 'Licence Responsible Officer', automatic: true },
+    ],
+    children: [{ groupCode: 'CHILD_1', groupName: 'Child - Site 1 - Group 2' }],
+  },
+}) =>
+  getFor({
+    urlPattern: '/groups/.*',
+    body: content,
+  })
```
Comment (karenmillermoj) [inline integration-tests/mockApis/manageusers.js:583]: We can probably reuse stubGroupDetailsWithChildren as it looks the same as this one

File: views/searchDpsUsers.njk [hunk 2]
```diff
@@ -188,7 +188,8 @@
 
                 {% set rows = [] %}
                 {% for u in results %}
-                  {% set maintainUserUrl = maintainUrl + '/' + u.username + '/details' %}
+                  {% set queryStr = '?searchType=beta&nomisEmail='+u.nomisEmailId if  u.nomisEmailId else '?searchType=beta' %}
+                  {% set maintainUserUrl = maintainUrl + '/' + u.username + '/details' + queryStr %}
                   {% set cellHtml %}
```
Comment (karenmillermoj) [inline views/searchDpsUsers.njk:193]: Is this change for the current PR ?

File: integration-tests/mockApis/manageusers.js (hunk 1 above)
Comment (ravmoj) [reply integration-tests/mockApis/manageusers.js]: removed stubManageUsersAssignableGroupDetails

File: views/searchDpsUsers.njk (hunk 2 above)
Comment (ravmoj) [reply views/searchDpsUsers.njk]: yes, removed this change

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**